src/components/SearchPage.vue

```diff
@@ -1,44 +1,51 @@
 <template>
-  <div class="grid-container body-search-content">
-    <PageHeading title="Symptoms Search"/>
-
-    <!-- search form -->
-    <form @submit.prevent>
-      <div class="grid-x grid-margin-x">
-        <div class="cell medium-8 large-6">
-          <SearchInput :filterCategories="selectedSystemIds" @itemSelected="handleSelection" :enableCategoryList="enableFilter"/>
-
-          <!-- annotation sufficiency information -->
-          <div class="quality-score">
-            <AnnotationSufficiency class="cell medium-8 large-6" :score="qualityScore" :error="scoringError"/>
-          </div>
-        </div>
-      </div>
+<div class="grid-container body-search-content">
+  <PageHeading title="Symptoms Search" />
+
+  <!-- search form -->
+  <form @submit.prevent>
+    <div class="grid-x grid-margin-x">
+      <div class="cell medium-8 large-6">
+        <SearchInput :filterCategories="selectedSystemIds" @itemSelected="handleSelection" :enableCategoryList="enableFilter" />
 
-      <!-- saved terms -->
-      <div class="grid-x grid-margin-x saved-terms">
-        <div class="cell">
-          <span v-for="(selection, index) in selections"
-                :key="selection.id"
-                class="label symptom-tag">{{ selection.symptomText }} <strong @click="removeSelection(index)">X</strong></span>
-        </div>
+      <!-- annotation sufficiency information -->
+      <div class="quality-score">
+        <AnnotationSufficiency :score="qualityScore" :error="scoringError" />
       </div>
+    </div>
 
-      <!-- buttons -->
-      <div class="grid-x grid-margin-x button-container">
-        <div class="cell large-4 text-left">
-          <input type="button" name="forwardButton" value="Done adding symptoms" class="button rounded" :disabled="selectionIsEmpty" @click="goForward">
-        </div>
+    <!-- About this search callouts -->
+    <div class="cell medium-3 large-3" v-if="!enableFilter">
+      <div class="callout success filter-search-msg">
+        <h6 class="title">About this Search</h6>
+        <p>Your search will be filtered by the categories you selected: <span class="filter-terms">{{selectedSystemLabels.join(', ')}}</span></p>
```

Comment:
This is probably okay, but I've been trying to avoid logic in the templates when I can implement an equivalent computed property.

Reply:
Oh, and this is already a computed property, so it's just a matter of moving the `join` call into the property.

Reply:
I'll rearrange it, but I had left it as a join in the template as I was considering a different treatment to the style. I'll put it back since we are going to hold off until we get feedback.